common-types/index.d.ts

```diff
@@ -168,3 +168,13 @@ interface DevPortfolioInfo {
 interface DevPortfolio extends DevPortfolioInfo {
   submissions: DevPortfolioSubmission[];
 }
+
+interface Shoutout {
+  readonly giver: IdolMember;
+  readonly receiver: string;
+  readonly message: string;
+  readonly isAnon: boolean;
+  readonly timestamp: number;
+  readonly hidden: boolean;
+  readonly uuid: string;
+}
```

Comment:
I agree withing moving `Shoutouts` to its proper place, but can we document this change in the PR since there's no associated ticket with it ?